packages/core/test/utils.test.ts

```diff
@@ -79,3 +92,86 @@ describe('Utils - getRoutes', () => {
     await expect(getRoutes(111, Environment.DEVNET)).rejects.toThrow('Invalid environment');
   });
 });
+
+describe('Address Validation Utils', () => {
+  describe('isValidSubstrateAddress', () => {
+    it('should validate a correct Substrate address', () => {
+      const validAddress = '1v72SmRT7XHwSa6hpEhUrgspM9bpmDRNw3F4wucb2GwkynQ';
+      expect(isValidSubstrateAddress(validAddress)).toBe(true);
+    });
+
+    it('should return false for an invalid Substrate address', () => {
+      const invalidAddress = 'invalidAddress';
+      expect(isValidSubstrateAddress(invalidAddress)).toBe(false);
+    });
+  });
+
+  describe('isValidEvmAddress', () => {
+    it('should validate a correct EVM address', () => {
+      const validAddress = '0x742d35Cc6634C0532925a3b844Bc454e4438f44e';
+      expect(isValidEvmAddress(validAddress)).toBe(true);
+    });
+
+    it('should return false for an invalid EVM address', () => {
+      const invalidAddress = 'invalidAddress';
+      expect(isValidEvmAddress(invalidAddress)).toBe(false);
+    });
+  });
+
+  describe('isValidBitcoinAddress', () => {
```

Comment:
for bitcoin we should add test cases for testnet. The problem here is the type of addresses that you have on btc, which could make this test suite somewhat bigger

Reply:
Added the tests